PIM/Integration/06_ManageCatalogs.md

```diff
@@ -0,0 +1,266 @@
+[!!User Interface Catalogs](../UserInterface/03f_Catalogs.md)
+[!!Manage the products](../Operation/01_ManageProducts.md)
+
+
+# Manage a catalog
+
+You can manage products in different catalogs and assign them to categories in order to sort the products and find them faster.
+
+[comment]: <> (UI will be changed)
+
+## Create a catalog
+
+Create a catalog with at least one category for new attributes you want to assign to a certain category.
+
+### Define the catalog basic data
+
+Create a new catalog and define its name to be able to create one or more categories afterwards.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+*PIM > Settings > Tab CATALOGS*
+
+![Catalogs](../../Assets/Screenshots/PIM/Settings/Catalogs/Catalogs.png "[Catalogs]")
+
+1. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the catalog name in the left column.   
+  The menu is displayed.
+
+  ![Menu](../../Assets/Screenshots/PIM/Settings/Catalogs/Menu.png "[Menu]")
+
+2. Click the *+ Create new* menu entry.   
+  The *Create new catalog* view is displayed.
+
+  ![Create new catalog](../../Assets/Screenshots/PIM/Settings/Catalogs/CreateNewCatalog.png "[Create new catalog]")
+
+3. Enter a name for the catalog in the text field in the header.    
+
+4. Continue to [add a category](#create-a-category) to the catalog, as at least one category per catalog is required to which you can assign the products.
+
+
+
+### Create a category
+
+You need to add at least one category to a catalog to be able to assign products to it.
+
+#### Prerequisites
+
+The catalog basic data has been created, see [Define the catalog basic data](#define-the catalog-basic-data).
+
+#### Procedure
+*PIM > Settings > Tab CATALOGS > Button Create new*
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  A plus sign is displayed in the category column.
+
+  ![Add category](../../Assets/Screenshots/PIM/Settings/Catalogs/AddCategory.png "[Add category]")
+
+2. Click the plus sign in the category column.    
+  The *Add Element* window is displayed in the category column.
+
+  ![Add element](../../Assets/Screenshots/PIM/Settings/Catalogs/AddElementTop.png "[Add element]")
+
+3. Enter a name for the category of the catalog in the *Name* field.
+
+4. Enter a key for the category of the catalog in the *Key* field.
+
+5. Click the [SAVE] button.   
+  The catalog category has been created. The *Add Element* window is closed. The new category is displayed in the category column.
+
```

Comment:
Wenn ich versuche, ein Catalog zu speichern (mit oder ohne category), bekomme die Fehlermeldung: Fields "key" could not be validated. Was mache ich falsch? Was muss man bei Key beachten? Evtl. Info dazu hinzufügen?